src/Catty/Helpers/Util/TableUtil.m

```diff
@@ -28,9 +28,18 @@
 
 @implementation TableUtil
 
+CGFloat statusBar;
+
++ (void) setStatusBar: (CGFloat) val
+{
+    statusBar = val;
+}
+
 + (CGFloat)heightForContinueCell:(CGFloat)navBarHeight
```

Comment:
Please do not store the `statusBar` as a static property in this util class which can be reused anywhere in the code. Out of context this is missleading, especially since this property needs to be set prior it can be used which requires your code in the `CatrobatTableViewController` to be executed before it can be used which is not always the case. To counteract this problem, add a second parameter, i.e. `statusBarHeight`, and do not store any further state in this util class.